systers_portal/templates/base.html

```diff
@@ -50,6 +50,12 @@
                    href="{% url 'new-community-proposal' %}">
                   New Community Proposal</a>
               </li>
+              {% if user.is_authenticated and user.is_active and user.is_staff %}
+              <li role="presentation">
+                <a role="menuitem" tabindex="-1" href="{% url 'add_community' %}">
```

Comment:
Why do you not want this not to be tabbed to? :)

Reply:
I am tabbing this because there's no separate page for communities, I think this would be easier for the admin to directly add a community, in the community section.

Reply:
I could have been more clear here, with `tabindex="-1"` user will not be able to tab to the this link, any specific reasons you think it should be the case.

Reply:
In the rest of the code in this file, `tabindex="-1"` is used , so I thought I should stick to it as well.

Reply:
I missed seeing other part. Can we change all then in this same pr?